packages/proxyIdentity/src/offerableIdentityOperator.ts

```diff
@@ -56,7 +56,8 @@ export class OfferableIdenitytOperator extends Operator {
 
   async offer(offerTo: string): Promise<boolean> {
     await this.identity.offer(
-      new Interface(identityAbi).functions.offer.encode([offerTo]),
+      // new Interface(identityAbi).functions.offer.encode([offerTo]),
```

Comment:
Probably this comment is no longer needed

Reply:
Yes indeed